master/static/migrations/20231010095640_fix-accelerator-data-table.tx.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE public.allocation_accelerators
+    DROP CONSTRAINT allocation_accelerators_pkey,
```

Comment:
I think you need to update the bun model with ```ID            int          `bun:"id,pk,autoincrement"````

Reply:
Done!